### PR TITLE
Retry only on CI

### PR DESCRIPTION
**Pull request by kiiadi**

### CODE REVIEW

**Review by rli — APPROVED** [review]

File: build.gradle.kts [hunk 1]
```diff
@@ -183,10 +183,12 @@ subprojects {
             html.isEnabled = true
         }
 
-        retry {
-            failOnPassedAfterRetry.set(false)
-            maxFailures.set(5)
-            maxRetries.set(2)
+        if (System.getenv("CI") != null) {
```
Comment (hunterwerlla) [inline build.gradle.kts:186]: There are 3 retry blocks (1 for each test type), please apply it to all 3/maybe make a function (probably not worth it)

Comment (kiiadi) [reply build.gradle.kts:186]: oh yeh - ok

File: build.gradle.kts [hunk 2]
```diff
@@ -303,7 +308,6 @@ intellij {
     version = ideProfile.community.sdkVersion
     pluginName = "aws-jetbrains-toolkit"
     updateSinceUntilBuild = false
-    downloadSources = System.getenv("CI") == null
```
Comment (hunterwerlla) [inline build.gradle.kts:306]: We download source on ci now?

**Review by hunterwerlla — APPROVED** [review]

**Review by abrooksv — APPROVED** [review]

**Pull request merged**